### ISSUE TITLE
MCXIVChordPlugin

### ISSUE THREAD

**Issue by Minecraftian14** [VCVRack/community#713]
Source Code: https://github.com/Minecraftian14/ChordPlugin

Given a scale and a note from that minor scale, this plugin gives three outputs corresponding to the chord played by that note in that scale.

For example, if the scale is C and the note is Eb, it will give the outputs Eb, G and Bb. If the note was F instead, it will output F, Ab and C. It automatically plays a minor or major chord where required.

**Comment by cschol** [comment]
@Minecraftian14 Why was this closed?

**Comment by Minecraftian14** [comment]
Dear @cschol,

It was my first c++ project, it was also my first VCV plugin. I took up this project as an adventure/experiment/fun. Yes, it was really fun making this! And as soon as it was working "slightly" as expected, I just went on and created this issue.

While it was being processed, I realized that project is stupid and incomplete.

* I learnt that it has (probably many) issues like invalid tags.
* It also adds along two stupid modules (which I created for testing).

What's next? Currently I'm busy with my end semester exams, after which, I'll remake the plugin with more functionality!

But, I'll not request this (and another plugin I made recently called [Mathobiz](https://github.com/Minecraftian14/Mathobiz/)) to be added to the VCV Library.

I feel, being an Open Source project, most of the ideas must have already been implemented one way or another. Next, I love math, but I'm not at all good with it. I don't even know most of the basics when it comes to music generation. 

It would have been really cool to have my plugin at VCV Library, but I think it's not eligible enough to be a part of that EPICNESS!

Thanks a lot - and sorry for creating this issue, I don't know how to delete this issue, "closing it" at the moment felt the best option.

Thank you.
